projects/canopy/src/lib/forms/sort-code/sort-code.stories.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormBuilder, FormGroup, ReactiveFormsModule } from '@angular/forms';
+
+import { moduleMetadata } from '@storybook/angular';
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+
+import { LgSortCodeComponent } from './sort-code.component';
+import { CanopyModule } from '../../canopy.module';
+import { notes } from './sort-code.notes';
+
+@Component({
+  selector: 'lg-reactive-form',
+  template: `
+    <form [formGroup]="form">
+      <lg-sort-code formControlName="sortCode">
+        {{ label }}
+        <lg-hint *ngIf="hint">{{ hint }}</lg-hint>
+      </lg-sort-code>
+    </form>
+  `,
+})
+class ReactiveFormComponent {
+  @Input() hint: string;
+  @Input() label: string;
+
+  @Input()
+  set disabled(disabled: boolean) {
+    if (disabled === true) {
+      this.form.controls.sortCode.disable();
+    } else {
+      this.form.controls.sortCode.enable();
+    }
+  }
+  get disabled(): boolean {
+    return this.form.controls.sortCode.disabled;
+  }
+
+  @Output() inputChange: EventEmitter<void> = new EventEmitter();
```

Comment:
this could be simplified to 

`@Output() inputChange = new EventEmitter<void>();`